site/source/tutorials/008_animations/index.md

```diff
@@ -0,0 +1,370 @@
+	this.invalidate();
+}
+
+//add to the top of the render function
+const zombieOneMoveAnimation = {
+	id: 'zombieOneMove,
+	effects: [
+		{ left: '0%' },
+		{ left: '35%' }
+	],
+	timing: {
+		duration: 8000,
+		easing: 'ease-in',
+		fill: 'forwards'
+	},
+	controls: {
+		play: true,
+		onFinish: this._onAnimationFinish // Add the callback here
+	}
+};
+```
+
+We will use this `onFinish` callback later in the tutorial to trigger a second animation.
+
+## Lets add more animations
+
+Not content with your zombie just sliding across the screen like that? Let's add some hollywood style move effects by animating the legs and the body.
+
+{% instruction 'Add more animations to Zombies.ts' %}
+
+``` typescript
+private _getZombieBodyAnimation(id: string): AnimationProperties {
+		return {
+			id,
+			effects: [
+				{ transform: 'rotate(0deg)' },
+				{ transform: 'rotate(-2deg)' },
+				{ transform: 'rotate(0deg)' },
+				{ transform: 'rotate(3deg)' },
+				{ transform: 'rotate(0deg)' }
+			],
+			timing: {
+				duration: 1000,
+				iterations: Infinity
+			},
+			controls: {
+				play: this._play
+			}
+		};
+	};
+
+	private _getZombieLegAnimation(id: string, front?: boolean): AnimationProperties {
+		const effects = [
+			{ transform: 'rotate(0deg)' },
+			{ transform: 'rotate(-5deg)' },
+			{ transform: 'rotate(0deg)' },
+			{ transform: 'rotate(5deg)' },
+			{ transform: 'rotate(0deg)' }
+		];
+
+		if (front) {
+			effects.reverse();
+		}
+
+		return {
+			id,
+			effects,
+			timing: {
+				duration: 1000,
+				iterations: Infinity
+			},
+			controls: {
+				play: this._play
+			}
+		};
+	}
+
+	// Add this to the render function with the other `animate` calls
+	webAnimation.animate('zombieOneBody', this._getZombieBodyAnimation('zombieOneBody'));
+	webAnimation.animate('zombieOneBackLeg', this._getZombieLegAnimation('zombieOneBackLeg'));
+	webAnimation.animate('zombieOneFrontLeg', this._getZombieLegAnimation('zombieOneFrontLeg', true));
+	webAnimation.animate('zombieTwoBody', this._getZombieBodyAnimation('zombieTwoBody'));
+	webAnimation.animate('zombieTwoBackLeg', this._getZombieLegAnimation('zombieTwoBackLeg'));
+	webAnimation.animate('zombieTwoFrontLeg', this._getZombieLegAnimation('zombieTwoFrontLeg', true));
+```
+
+Refresh your browser and you should now see two stumbling zombies moving towards one another. Feel free to have a play with the params for rotation / timings to get it just how you like it.
+
+![Zombie-Walk](./walking.gif)
+
+Notice that these two new animations have `iterations` set to `Infinity`. This ensures that they will play indefinitely, or until the first animation completes and `this._play` is set to false.
+
+{% section %}
+
+## Joining Animations
+
+Lets finish off by adding some hearts that appear once the Zombies have reached the middle of the screen. We want them to float upto the top of the screen whilst changing size. We can do this by returning an array of `AnimationProperties` rather than a single object.
```

Comment:
Lets finish off by adding some hearts that appear once the Zombies have reached the middle of the screen. We want them to float upto the top of the screen whilst changing size. We can do this by returning an array of `AnimationProperties` rather than a single object. -> Let's finish by adding some hearts that appear once the Zombies have reached the middle of the screen. We want them to float up to the top of the screen while changing size. We can do this by returning an array of `AnimationProperties` rather than a single object.